### PR TITLE
docs: authentication, UI5 obj API alignment, wdi5.url deprecation, and more

### PR DESCRIPTION
**Pull request by vobu**

### CODE REVIEW

File: docs/recipes.md [hunk 1]
```diff
@@ -340,3 +315,40 @@ it("should find the search button on a SearchField", async () => {
 ```
 
 ?> Note: More info on this see: https://github.com/SAP/ui5-uiveri5/blob/master/docs/usage/locators.md#interaction-adapters
+
+## using wdio functions
+
+WebdriverIO has a extensive element [API](https://webdriver.io/docs/api/). The [Element API] specifically can be quite useful to check if the page elements are in a certain state eg. [isDisplayed](https://webdriver.io/docs/api/element/isDisplayed) or [isClickable](https://webdriver.io/docs/api/element/isClickable).
```
Comment (Siolto) [inline docs/recipes.md:321]:
```suggestion
WebdriverIO has a extensive element [API](https://webdriver.io/docs/api/). The [Element API] specifically can be quite useful to check if the page elements are in a certain state e.g. [isDisplayed](https://webdriver.io/docs/api/element/isDisplayed) or [isClickable](https://webdriver.io/docs/api/element/isClickable).
```

File: docs/usage.md [hunk 2]
```diff
@@ -2,7 +2,36 @@
...
+// assuming title is a sap.m.Title
+const title = await browser.asControl(selector)
+// bindingInfo is a UI5 Managed Object, not a UI5 control!
+const bindingInfo = await title.getModel()
+const name = await bindingInfo.getProperty("/path/in/model") // same .getProperty in wdi5 and UI5!
+```
+
+?> there is no [fluent async api](#fluent-async-api) available for both the aligned Managed Object API and [`.asObject`](#asobject)
+
+At the same time, the `wdi5`-api can be mixed with `wdio`'s api during tests at will - there is no restriction to use either or. Even a [fluent async transition](ttps://ui5-community.github.io/wdi5/#/recipes?id=using-wdio-functions) between the two is possible:
```
Comment (Siolto) [inline docs/usage.md:25]:
```suggestion
At the same time, the `wdi5`-api can be mixed with `wdio`'s api during tests at will - there is no restriction to use either one. Even a [fluent async transition](ttps://ui5-community.github.io/wdi5/#/recipes?id=using-wdio-functions) between the two is possible:
```

**Review by Siolto — CHANGES_REQUESTED** [review]
just very small remarks. The rest is very well written in my opinion 👍

**Review by dominikfeininger — APPROVED** [review]

File: docs/authentication.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+    // "two" is the literal reference to a browser instance
+    two: {
+        capabilities: {
+            "wdi5:authentication": {
+                provider: "BTP" // "Office365" | "custom" | "BasicAuth"
+            },
+            // ...
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (vobu) [reply docs/authentication.md:68]: shortened the example

**Review by Siolto — APPROVED** [review]

**Pull request merged**